configs/README.MD

```diff
@@ -0,0 +1,32 @@
+# Configuration
+
+The [configs](../master/configs/configs.csv) file contains a list of configuration commands for any number of cameras listed in the [paths](../master/cameras/paths.csv) file.
+
+
+## Fields
+
+| Field | Description | Required |
+| ----- | ----------- | -------- |
+| maker	| The name of the camera manufacturer, e.g. "Sharx" or "Amcrest" | required |
+| command_id	| An unique identifier string for the command, e.g. "enable_wifi" | required |
+| url	| The full url used to execute the camera configuration | required |
+| method | The method used to access `url`, e.g. "GET" | required |
+| regex_model	| A regex representing all the models that support the comamnd, e.g "SCNC3[89]0[45]" | required |
+| regex_firmware	| A regex representing all the firmware versions that support the comamnd, e.g "^v4" | required |
```

Comment:
`auth_type` as optional.
The camera's server *should* reply with `WWW-Authenticate: Digest` to tell us that we should try `Authorization: Digest` , but I guess we can include this optional value for now. 
We may need to re-think this and just automatically try `Authorizaiton: Digest ...` whenever `Authorization: Basic ...` fails. So we may eliminate this field.